frontend/src/queries/nodes/DataTable/ColumnConfigurator/ColumnConfigurator.tsx

```diff
@@ -42,10 +42,16 @@ export function ColumnConfigurator({ query, setQuery }: ColumnConfiguratorProps)
         columns: columnsInQuery,
         setColumns: (columns: string[]) => {
             if (isEventsQuery(query.source)) {
+                const oldOrderBy = query.source.orderBy
+                const firstOldOrderBy = oldOrderBy?.length == 1 ? removeExpressionComment(oldOrderBy?.[0]) : undefined
```

Comment:
```suggestion
                const firstOldOrderBy = oldOrderBy?.length === 1 ? removeExpressionComment(oldOrderBy?.[0]) : undefined
```

Reply:
Quite hard to understand what is going on here, but no real suggestion for improvement.

Reply:
did something, hope that's clearer :)